### PR TITLE
Add DueFocus 2.0.8

### PR DESCRIPTION
**Pull request by VadimKorobka**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/duefocus.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'duefocus' do
+  version '2.0.8'
+  sha256 'd14b26054bceb347ac6d94945e5215fb030f1f7447e012d86d35feadb677d8e6'
+
+  url "https://web.duefocus.com/distribution/darwin/v3/DueFocus-#{version}-mac.zip"
```
Comment (vitorgalvao) [inline Casks/duefocus.rb]: Where did you find this download?

Comment (KorobkaVadim) [reply Casks/duefocus.rb]: I'm a developer of this product.

Comment (vitorgalvao) [reply Casks/duefocus.rb]: Why do you provide a `pkg` on the webpage?

And can you give us an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

Comment (KorobkaVadim) [reply Casks/duefocus.rb]: It is advisable for us to show the license agreement during installation. When we had a choice between `pkg` and `dmg`, the ability to add a license to the `dmg` file was absent in the electronic builder.

Comment (vitorgalvao) [reply Casks/duefocus.rb]: What about the appcast part of the question? That’s more important.

Comment (KorobkaVadim) [reply Casks/duefocus.rb]: I will research the question regarding appcast, since it is not entirely clear what it is and where to get it.

Comment (vitorgalvao) [reply Casks/duefocus.rb]: It’s a file or a webpage hosted somewhere that always has a reference to the latest version of an app.

Comment (KorobkaVadim) [reply Casks/duefocus.rb]: @vitorgalvao fixed

Comment (vitorgalvao) [reply Casks/duefocus.rb]: That works. Thank you.

**Pull request merged**